metadata/src/main/java/org/apache/kafka/controller/PeriodicTask.java

```diff
@@ -43,6 +50,8 @@ class PeriodicTask {
      */
     private final EnumSet<PeriodicTaskFlag> flags;
 
+    private static final long IMMEDIATE_PERIOD_NS = MILLISECONDS.toNanos(10);
```

Comment:
Can you add a new constructor that allows setting this? You can leave the old constructor with this as the default, of course.